src/app/pages/proof/proof.page.ts

```diff
@@ -48,6 +49,18 @@ export class ProofPage {
       }));
     })
   );
+
+  readonly information$ = this.proof$.pipe(
```

Comment:
Unnecessary property. See the [comment](https://github.com/numbersprotocol/capture-lite/pull/52/files#diff-1076f52a952e84cba221250338e0c01dR61).